### ISSUE TITLE
French formatting of dates in moment

### ISSUE THREAD

**Issue by madjam002** [wanasit/chrono#198]
Since 1.3.2 if I format dates using the localized formats available in `moment`, the dates appear to be following the French locale.

I imagine this is due to NPM flattening dependencies so the same `moment` is being used by my project and the `chrono-node` dependency, and `chrono-node` is defining a French locale which is somehow overriding the global default?

**Comment by wanasit** [comment]
Interesting. Maybe something related to [this change](https://github.com/wanasit/chrono/commit/ac2c6d8ddc751f5fef4e0c8b0277a8c80c331b32#diff-646bd581dbd6e7837d3bf6945786b105). 

Will look into this.

**Comment by mycarrysun** [comment]
I'm getting the same thing...is there a temporary workaround to force the English locale in the meantime?

**Comment by pepoviola** [comment]
I get the same error, rollback to use `1.3.1` works ok for me. Force moment to use English locale after import chrono-node ( `moment.locale('en');` ) also works.

**Comment by wanasit** [comment]
Look like including FR locale in moment also changed the global locale. Found this code in moment.js
```
function loadLocale(name) {
    var oldLocale = null;
    // TODO: Find a better way to register and load all the locales in Node
    if (!locales[name] && (typeof module !== 'undefined') &&
            module && module.exports) {
        try {
            oldLocale = globalLocale._abbr;
            require('./locale/' + name);
            // because defineLocale currently also sets the global locale, we
            // want to undo that for lazy loaded locales
            getSetGlobalLocale(oldLocale);
        } catch (e) { }
    }
    return locales[name];
}
```
The quick fix in de8ee9a should work. Please upgrade to 1.3.3.
